src/Riprap/IslandoraRiprapUtils.php

```diff
@@ -71,8 +71,13 @@ public function getFileUuid($mid) {
   public function getFedoraUrl($mid) {
     $media = Media::load($mid);
```

Comment:
Any reason we can't put the following check here, right after loading the media?

```php
if ($media->bundle() == 'remote_video') {
   return False
}
```
It's a bit simpler to read, and returns before executing the next two lines.

Reply:
will do -- very soon I will refactor this method to handle the issue I was seeing.